### ISSUE TITLE
[Dropdown] How to set the input to only search but not filter options?

### ISSUE THREAD

**Issue by peili0413** [fomantic/Fomantic-UI#1241]
What i want is when the user type something in the dropdown's input, a request send and dynamic generate the options, and the options should always showing even they don't match the input's value.

For example, user can input `lv`, and the `valuesFromResponse(data)` is `[{value: 'lenovo', text: 'lenove', name: 'lenove/thinkpad/x1'}, ...]`. But the problem is, the input's value `lv` filter the result so options are not showing. How should i fix this problem?

ajax request
```javascript
$.ajax({
    ...
    success: function(data) {
        $('.ui.dropdown').dropdown('setup menu', {
            values: valuesFromResponse(data)
        })
    }
})
```

html
```html
<div class="field">
    <select class="ui search dropdown">
    </select>
</div>
```

.ui.dropdown settings
```javascript
// settings
{
sortSelect: true,
fullTextSearch: 'exact',
on: 'hover',
allowAdditions: true,
clearable: true,
forceSelection: false
}
```

**Comment by lubber-de** [comment]
This should be possible using the fresh released 2.8.3.
Add the class `unfilterable` to each item. So your response should look like

```javascript
[{
  value: 'lenovo',
  text: 'lenove',
  name: 'lenove/thinkpad/x1',
  class: 'unfilterable'
}]
```

I have to admit, we forgot to put that feature into the docs... 🙄

**Comment by peili0413** [comment]
I solved the problem by using
```javascript
$('.ui.dropdown').dropdown('show')
```
Your solution is more concise. Thanks for your diligent update.
